modules/ROOT/pages/_partials/concepts/sync-function.adoc

```diff
@@ -39,53 +39,70 @@ Any validation checks will probably fail because of the missing properties, so b
 
 :note-for-prototype: [This prototype shows the default Sync Function -- see <<lbl-args>> for more on the arguments.]
 
-== Sync Function Prototype
+[#ex-sync-function]
+== Sync Function Examples
 
-[#ex-prototype]
-.Prototype Sync Function
+Couchbase Sync Gateway defines a Sync Function at the `collection` level. 
+Defining at this level helps simplify data management and improve data reliability. 
+Each collection in the system allows for only one Sync Function, which enables the specification of Access Control rules.
+
+.Default Sync Function
 ====
 
-[{tabs}]
```

Comment:
Just want to check why you got rid of the tabs here?
Also, any oddities from the successive `====` and `=====` blocks now?